.github/actions/setup-deps/action.yaml

```diff
@@ -78,6 +78,8 @@ inputs:
     default: 'seaborn>=0.7.0'
   tidynamics:
     default: 'tidynamics>=1.0.0'
+  dask:
```

Comment:
Can you move this to above distopia please? These are meant to be alphabetically ordered